Bio/SwissProt/__init__.py

```diff
@@ -447,11 +453,15 @@ def _read_rn(reference, rn):
     # RN   [1] {ECO:0000313|EMBL:AEX14553.1}
     words = rn.split(None, 1)
     number = words[0]
-    assert number.startswith("[") and number.endswith("]"), "Missing brackets %s" % number
+    assert number.startswith("[") and number.endswith("]"), (
+        "Missing brackets %s" % number
+    )
```

Comment:
I don't think we have an issue for this exactly, #2015 was close in nature. Ideally these asserts would be replaced with if statements and ``raise ValueError`` instead.